AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -448,4 +454,24 @@ - (void)applicationDidEnterBackground {
 }
 #endif
 
+#pragma mark - Disable services for test cloud
+
+/**
+ * Determines whether a service should be disabled.
+ *
+ * @param serviceName The service name to consider for disabling.
+ *
+ * @return YES if the service should be disabled.
+ */
+- (BOOL)shouldDisable:(NSString*)serviceName {
+  NSDictionary *environmentVariables = [[NSProcessInfo processInfo] environment];
+  NSString *disabledServices = environmentVariables[@"APP_CENTER_DISABLE"];
+  if (!disabledServices) {
+    return NO;
+  }
+  NSArray* disabledServicesList = [disabledServices componentsSeparatedByString:@","];
```

Comment:
Can you also trim the items in the list? Just adding one more protection if the environment variable has spaces.

Reply:
Do you mean whitespace at the beginning/end? or even in the middle?

Reply:
My concern is that if the value is provided like `Something, Something, Something`.
In this case, you will have `Something`, `<space>Something` and `<space>Something`.
I don't think you need to take care of trimming all spaces at the end or in the middle of strings but beginning should be.